### ISSUE TITLE
Crash: NSInvalidArgumentException *** -[__NSArrayM insertObject:atIndex:]: object cannot be nil

### ISSUE THREAD

**Issue by alex-ost** [facebook/facebook-ios-sdk#1162]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

- Xcode Version: `11.3`
- Swift Version: `5`
- Installation Platform & Verison: `[Cocoapods]` version `1.8.3`

## Report

I see random crashes related to FBSDKCoreKit, description:
```
Fatal Exception: NSInvalidArgumentException
*** -[__NSArrayM insertObject:atIndex:]: object cannot be nil

```
## Steps to Reproduce

No idea. I have this crash reported from Crashlytics. 

## Crashing call stack

```
0 | CoreFoundation | __exceptionPreprocess
-- | -- | --
1 | libobjc.A.dylib | objc_exception_throw
2 | CoreFoundation | -[__NSCFString characterAtIndex:].cold.1
3 | CoreFoundation | -[__NSArrayM insertObject:atIndex:].cold.2
4 | CoreFoundation | -[__NSArrayM insertObject:atIndex:]
5 | FBSDKCoreKit | FBSDKCrashHandler.m line 217+[FBSDKCrashHandler loadCrashLogs] + 217
6 | FBSDKCoreKit | FBSDKCrashHandler.m line 185+[FBSDKCrashHandler getProcessedCrashLogs] + 185
7 | FBSDKCoreKit | FBSDKCrashHandler.m line 114__33+[FBSDKCrashHandler addObserver:]_block_invoke + 114
8 | libdispatch.dylib | _dispatch_client_callout
9 | libdispatch.dylib | _dispatch_once_callout
10 | FBSDKCoreKit | once.h line 84+[FBSDKCrashHandler addObserver:] + 84
11 | FBSDKCoreKit | FBSDKCrashObserver.m line 48+[FBSDKCrashObserver enable] + 48
12 | FBSDKCoreKit | FBSDKGateKeeperManager.m line 91+[FBSDKGateKeeperManager loadGateKeepers:] + 91
13 | FBSDKCoreKit | FBSDKFeatureManager.m line 43+[FBSDKFeatureManager checkFeature:completionBlock:] + 43
14 | FBSDKCoreKit | FBSDKInstrumentManager.m line 39+[FBSDKInstrumentManager enable] + 39
15 | FBSDKCoreKit | FBSDKGateKeeperManager.m line 181+[FBSDKGateKeeperManager _didProcessGKFromNetwork:] + 181
16 | FBSDKCoreKit | FBSDKGateKeeperManager.m line 172+[FBSDKGateKeeperManager processLoadRequestResponse:error:] + 172
17 | FBSDKCoreKit | FBSDKGateKeeperManager.m line 104__42+[FBSDKGateKeeperManager loadGateKeepers:]_block_invoke + 104
18 | FBSDKCoreKit | FBSDKGraphRequestMetadata.m line 41-[FBSDKGraphRequestMetadata invokeCompletionHandlerForConnection:withResults:error:] + 41
19 | FBSDKCoreKit | FBSDKGraphRequestConnection.m line 800__82-[FBSDKGraphRequestConnection processResultBody:error:metadata:canNotifyDelegate:]_block_invoke + 800
20 | FBSDKCoreKit | FBSDKGraphRequestConnection.m line 833-[FBSDKGraphRequestConnection processResultBody:error:metadata:canNotifyDelegate:] + 833
21 | FBSDKCoreKit | FBSDKGraphRequestConnection.m line 782__64-[FBSDKGraphRequestConnection completeWithResults:networkError:]_block_invoke + 782
22 | CoreFoundation | __NSARRAY_IS_CALLING_OUT_TO_A_BLOCK__
23 | CoreFoundation | -[__NSArrayM enumerateObjectsWithOptions:usingBlock:]
24 | FBSDKCoreKit | FBSDKGraphRequestConnection.m line 761-[FBSDKGraphRequestConnection completeWithResults:networkError:] + 761
25 | FBSDKCoreKit | FBSDKGraphRequestConnection.m line 628-[FBSDKGraphRequestConnection completeFBSDKURLSessionWithResponse:data:networkError:] + 628
26 | FBSDKCoreKit | FBSDKGraphRequestConnection.m line 241__36-[FBSDKGraphRequestConnection start]_block_invoke_3 + 241
27 | libdispatch.dylib | _dispatch_call_block_and_release
28 | libdispatch.dylib | _dispatch_client_callout
29 | libdispatch.dylib | _dispatch_main_queue_callback_4CF$VARIANT$mp
30 | CoreFoundation | __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
31 | CoreFoundation | __CFRunLoopRun
32 | CoreFoundation | CFRunLoopRunSpecific
33 | GraphicsServices | GSEventRunModal
34 | UIKitCore | UIApplicationMain
35 | MyApp | AppDelegate.swift line 38main + 38
36 | libdyld.dylib | start
```

**Comment by philfi** [comment]
Experiencing the same crash, I'm curious if you're using any other crash reporting libraries? I've been thinking it might be a conflict with another crash reporting library we have in our app.

**Comment by KylinChang** [comment]
Hi @philfi , which crash reporting libraries are you using?

**Comment by philfi** [comment]
@KylinChang Just Crashlytics at current

**Comment by tianqibt** [comment]
Thanks for reporting the issue! We have delivered the fix and will release a new version soon!

**Comment by hissain** [comment]
Is there any update about that issue? @tianqibt

**Comment by tianqibt** [comment]
The fix is released with v5.15.1.
